src/document_array_clear.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from jina import Document, DocumentArray
+from jina.types.arrays.memmap import DocumentArrayMemmap
+
+from .utils.benchmark import benchmark_time
+
+NUM_REPETITIONS = 10
+
+
+@pytest.mark.parametrize('num_docs', [100, 1000, 10_000])
```

Comment:
typo?
```suggestion
@pytest.mark.parametrize('num_docs', [100, 1000, 10000])
```

Reply:
Not a typo  https://www.python.org/dev/peps/pep-0515/ , I think it makes numbers with many zeros much more readable.